### PR TITLE
Apply build updates, switch to 6.1.1 of the settings plugin and update build

### PR DESCRIPTION
**Pull request by timyates**

### CODE REVIEW

**Comment by timyates** [comment]
Ok, so I think this is failing due to a change in liquibase

https://docs.liquibase.com/concepts/changelogs/how-liquibase-finds-files.html

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,21 +1,25 @@
 [versions]
-managed-liquibase = "4.17.2"
+managed-liquibase = "4.18.0"
```
Comment (wetted) [inline gradle/libs.versions.toml:2]: if this is a breaking change, back it out and consider working it separately from the renovate PR

**Review by wetted — APPROVED** [review]

File: gradle/libs.versions.toml (hunk 1 above)
Comment (timyates) [reply gradle/libs.versions.toml]: The previous version doesn't work either, so I reverted and then went further back to the version in the last release (v5.5.0)

File: liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetWrapper.java [hunk 2]
```diff
@@ -58,9 +58,8 @@ public Set<String> getContexts() {
         return delegate.getContextExpression().getContexts();
     }
 
-    @JsonFormat(shape = JsonFormat.Shape.STRING)
```
Comment (timyates) [inline liquibase/src/main/java/io/micronaut/liquibase/endpoint/RanChangeSetWrapper.java:61]: This doesn't exist for serde, so I did it by hand

File: gradle/libs.versions.toml [hunk 3]
```diff
@@ -1,21 +1,25 @@
 [versions]
-managed-liquibase = "4.17.2"
+managed-liquibase = "4.16.0"
```
Comment (timyates) [inline gradle/libs.versions.toml:2]: 4.16.0 is the version we released with 5.5.0

Versions 4.17+ don't work as they changed how changesets are looked up and loaded

https://docs.liquibase.com/concepts/changelogs/how-liquibase-finds-files.html

**Pull request merged**